web/client/components/leaflet/Map.jsx

```diff
@@ -7,6 +7,11 @@
  */
 var L = require('leaflet');
 var React = require('react');
+var ScaleBox = require('../../api/ScaleBox');
```

Comment:
Map should not depend on optional components. Components inclusion should be done by composition on a higher level container.